src/librbd/cache/pwl/SSDWriteLog.cc

```diff
@@ -77,7 +77,7 @@ void SSDWriteLog<I>::initialize_pool(Context *on_finish, pwl::DeferredContexts &
     m_cache_state->clean = true;
     m_cache_state->empty = true;
     /* new pool, calculate and store metadata */
-    size_t small_write_size = MIN_WRITE_ALLOC_SIZE + sizeof(struct WriteLogPmemEntry);
+    size_t small_write_size = MIN_WRITE_ALLOC_SSD_SIZE + sizeof(struct WriteLogPmemEntry);
```

Comment:
Nit: weird seeing any class named `Pmem` in the SSD-version

Reply:
`WriteLogPmemEntry` is the on-disk data structure that is shared by both RWL and SSD versions. We could change the name to `WriteLogDiskEntry` if that sounds ok.